automatic/prtgdesktop/prtgdesktop.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>prtgdesktop</id>
+    <title>PRTG Desktop</title>
+    <version>20.8.0</version>
+    <authors>Paessler AG</authors>
+    <owners>BBT Software AG</owners>
+    <projectUrl>https://www.paessler.com/prtg-desktop-app</projectUrl>
+    <packageSourceUrl>https://github.com/bbtsoftware/chocolatey-packages/tree/master/automatic/prtgdesktop</packageSourceUrl>
+    <iconUrl>https://cdn.jsdelivr.net/gh/bbtsoftware/chocolatey-packages@1e64724bec93a0e22bb17e7dbd66f132bf106527/icons/prtgdesktop.png</iconUrl>
+    <docsUrl>https://manuals.paessler.com/prtgdesktop/prtgdesktop.pdf</docsUrl>
```

Comment:
Isn't there a website providing documentation? I wouldn't expect to download a PDF when clicking on documentation link on chocolatey.org.

Reply:
Removed, since there is no specific docs URL